recipes/humap/meta.yaml

```diff
@@ -0,0 +1,63 @@
+{% set name = "humap" %}
+{% set version = "0.2.3" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: e9f59bd34fc7b7834010ab91dddcd83af53812af2b91b2973924ee474bf110d2
+  
+build:
+  script:
+    - python setup.py build_ext -I$PREFIX/include/eigen3 install  # [osx]
+    - python setup.py build_ext -I$PREFIX/include/eigen3 install  # [linux]  
+    - python setup.py build_ext -I%PREFIX%\Library\include\eigen3 install  # [win]
+  number: 0
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+  host:
+    - libgomp  # [linux]
+    - numpy ==1.23.0
+    - scipy ==1.9.3
+    - scikit-learn ==1.1.3
+    - eigen ==3.4.0
+    - pybind11 ==2.10.1
+    - llvm-openmp >=14.0.4  # [osx]
```

Comment:
What is the reason for this very tight pins? The original `setup.py` doesn't seem to have them: https://github.com/wilsonjr/humap/blob/69da0c438e2f4b2544fc9224010beacda19f174b/setup.py#L70

Reply:
Hi @xhochy, this is an attempt to solve the conflicts when running the pipeline. I'll update the the recipe to make it >=versions. At least for eigen and pybind11 I need these versions.

Reply:
@xhochy can you revise, please? It turns out I need the march=native for compiling a few cpp files that perform distance calculations.

Reply:
No, `march=native` is a no-go on conda-forge. This will produce non-portable binaries and thus make the package mostly useless.

Reply:
@xhochy, do you know the alternative for this flag?

Reply:
No, you have to wait for an alternative reviewer. That's outside of my knowledge. cc @conda-forge/help-c-cpp

Reply:
hi @xhochy, can you please revise? I updated the code and removed cpu specific snippets.